### ISSUE TITLE
Run Check if Forgot Password link is clicked for Institutional Accounts

### ISSUE THREAD

**Issue by cbwinchild** [Tizra/Tizra-Customer-Tracker#351]

**Comment by adane** [comment]
Do you mean IP authenticated accounts?  We've discussed some approaches to preventing nuisance password changes on shared password accounts (such as classroom accounts).

**Comment by cbwinchild** [comment]
The enhancement should have been called: Run Check if Forgot Password link
is clicked for Institutional Accounts, so if user clicks a Forgot Password
link, they won't change the PW for an institutional user that has a shared
username/PS. I changed the name of hte enhancemt. Does this make sense?

On Fri, Jun 20, 2014 at 3:40 PM, Abe Dane notifications@github.com wrote:

> Do you mean IP authenticated accounts? We've discussed some approaches to
> preventing nuisance password changes on shared password accounts (such as
> classroom accounts).
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Tizra/Tizra-Customer-Tracker/issues/351#issuecomment-46718511
> .

## 

Catherine B. Winchild
Office Hours: M-F 10:30am - 2:30pm EST
Technical Writer & Trainer
Tizra, Inc.
9 Catalpa Rd.
Providence, RI 02906
_Phone:_ +1 (855) 220-1235
_Fax:_+1 (401) 273-5171

**Comment by adane** [comment]
Yes, we’d talked about handling this by having the forgot password form send the user a link.  The link would lead to a form where the password could be re-set.  If the password is not re-set within a certain period of time (potentially because the request was bogus) then the password would remain unchanged.

On Jun 20, 2014, at 4:30 PM, cbwinchild notifications@github.com wrote:

> The enhancement should have been called: Run Check if Forgot Password link 
> is clicked for Institutional Accounts, so if user clicks a Forgot Password 
> link, they won't change the PW for an institutional user that has a shared 
> username/PS. I changed the name of hte enhancemt. Does this make sense? 
> 
> On Fri, Jun 20, 2014 at 3:40 PM, Abe Dane notifications@github.com wrote: 
> 
> > Do you mean IP authenticated accounts? We've discussed some approaches to 
> > preventing nuisance password changes on shared password accounts (such as 
> > classroom accounts). 
> > 
> > — 
> > Reply to this email directly or view it on GitHub 
> > https://github.com/Tizra/Tizra-Customer-Tracker/issues/351#issuecomment-46718511 
> > . 
> 
> ## 
> 
> Catherine B. Winchild 
> Office Hours: M-F 10:30am - 2:30pm EST 
> Technical Writer & Trainer 
> Tizra, Inc. 
> 9 Catalpa Rd. 
> Providence, RI 02906 
> _Phone:_ +1 (855) 220-1235 
> _Fax:_+1 (401) 273-5171
> —
> Reply to this email directly or view it on GitHub.

## 

Abe Dane
President and COO
Tizra, Inc.
web: http://tizra.com
blog: http://blog.tizra.com
twitter: http://twitter.com/tizra

abe.dane@tizra.com
v: 401 787-6427
f:  401 273-5171
